api/config/base/rbac/role.yaml

```diff
@@ -23,6 +23,9 @@ rules:
   - get
   - list
   - watch
+  - create
```

Comment:
Are these generated somewhere else?

Reply:
I just realized when I run `make build-reference-api` it deletes them.

Reply:
Add them as kubebuilder annotations, you'll be fine

Reply:
Thanks I added them to the apps repo and pushed a new commit! 👍